src/models/src/events/interaction.rs

```diff
@@ -117,6 +117,7 @@ pub struct SlackInteractionViewSubmissionEvent {
     pub team: SlackBasicTeamInfo,
     pub user: SlackBasicUserInfo,
     pub view: SlackStatefulView,
+    pub trigger_id: Option<SlackTriggerId>,
```

Comment:
`trigger_id` fields are actually exists always, but keeping in Option may be needed 'cause this is not documented tho.